drivers/bluetooth/hci/ipc.c

```diff
@@ -22,6 +22,10 @@ LOG_MODULE_REGISTER(bt_hci_driver);
 
 #define IPC_BOUND_TIMEOUT_IN_MS K_MSEC(1000)
 
+#define USE_SLEEP_BETWEEN_IPC_RETRIES COND_CODE_0(CONFIG_SYS_CLOCK_TICKS_PER_SEC, \
+	(0), \
+	((USEC_PER_SEC / CONFIG_SYS_CLOCK_TICKS_PER_SEC) > CONFIG_BT_HCI_IPC_SEND_RETRY_DELAY_US))
```

Comment:
```suggestion
#define USE_SLEEP_BETWEEN_IPC_RETRIES COND_CODE_0(CONFIG_SYS_CLOCK_TICKS_PER_SEC, \
	(false), \
	((USEC_PER_SEC / CONFIG_SYS_CLOCK_TICKS_PER_SEC) > CONFIG_BT_HCI_IPC_SEND_RETRY_DELAY_US))
```

Minor suggestion to use `bool` values

Reply:
Changed